src/commands/react-native/__tests__/codepush.test.ts

```diff
@@ -46,10 +46,6 @@ jest.mock('child_process', () => ({
   }),
 }))
 
-beforeEach(() => {
-  jest.restoreAllMocks()
-})
```

Comment:
The new version of jest changed the behavior of `restoreAllMocks` (https://github.com/jestjs/jest/issues/13925).

It now breaks the global `jest.mock()` above. Turns out this `beforeEach()` isn't required to make the tests pass.